Sources/PrivoSDK/internal/age-gate/PrivoAgeHelpers.swift

```diff
@@ -6,6 +6,7 @@
 //
 
 import Foundation
+import Alamofire
```

Comment:
why are you moving Alamofire and network-check outside of the Network layer?

Reply:
Okay. Removed the alamofire import here.

The main reason to move out checkNetwork from the Rest class due to these points:
- method checkNetwork is not tied to Rest instance data really
-  privo age helper instance have a dependency to rest fictitious instance 

Anyway, improved decision and delete this flashing dependency import.